File: 프로그래머스/unrated/181938. 두 수의 연산값 비교하기/두 수의 연산값 비교하기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int a, int b) {
    int answer = 0;
    int a_b = stoi(to_string(a) + to_string(b));
    int mul_a_b = 2*a*b;
    return a_b > mul_a_b? a_b : mul_a_b;
}

File: 프로그래머스/unrated/181938. 두 수의 연산값 비교하기/두 수의 연산값 비교하기.java
class Solution {
    public int solution(int a, int b) {
        int answer = 0;
        int a_b = Integer.valueOf(String.valueOf(a) + String.valueOf(b));
        int mul_a_b = 2*a*b;
        if(a_b < mul_a_b) return mul_a_b;
        return a_b;
    }
}